Lesson3.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace Lesson3
+{
+    public class Lesson3
+    {
+	    public Lesson3()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4971681)

Reply:
So should I not use tab in C#? Also, is there an automated way to do that?

Reply:
No, it's a weird issue. I will remove it from scan, please ignore.